aten/src/ATen/native/ForeachUtils.h

```diff
@@ -128,10 +128,29 @@ inline bool _check_tensors_share_device_and_dtype(
 // corresponding tensors in tensor lists have the same sizes and strides.
 inline bool _check_tensors_share_sizes_and_strides(
     ArrayRef<TensorList> tensorLists) {
+  auto is_same_stride = [](const IntArrayRef& size,
+                           const IntArrayRef& left_stride,
+                           const IntArrayRef& right_stride) -> bool {
+    const size_t size_size = size.size();
+    if (left_stride.size() != size_size || right_stride.size() != size_size) {
+      return true;
```

Comment:
shouldn't this be false?

also isn't left_stride.size() always equal to size_size?

Reply:
yep, tensor sizes compared before, and here no need to check tensor sizes length.